Week1/homework/4-Log the string.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
```

Comment:
Did you try to run this code? I think it will throw an error.

Reply:
i don't know what is the problem

Reply:
it's working